doc/Security.xml

```diff
@@ -4861,30 +4861,6 @@
       <para>All other configuration of the TLS server on a device is outside the scope of this specification revision and may be addressed by later revisions of this document.</para>
     </section>
   </chapter>
-  <appendix xml:id= "q5l_q5m_51c">
-    <title>JWT Content example</title>
-    <para>Here is an example of JWT components to be encoded into a JWT Token</para>
-    <programlisting><![CDATA[{
-    "typ":"JWT",
-    "alg":"ES256",
-}.{
-    "iss":"server.path.com", // FQDN of the server that issued the JWT
-    "nbf":1670482800, // not before Thursday, December 8, 2022 7:00:00 AM (GMT)
```

Comment:
please add JWT content example which match with JWT token mentioned in SCTP, RTSPS example section below instead of completely removing it.